### ISSUE TITLE
Feature Request: New Manually Geofencing Mode (Auto - Home - Away)

### ISSUE THREAD

**Issue by Nastras** [SeydX/homebridge-tado-platform#39]
Hello @SeydX,
there is a new feature in the current beta of the Tado app. With the new function it is possible to change the presence mode (geofencing) manually:

Automatic - Home - Away

This function is very useful for a guest mode or similar.
Would be nice if you could implement this function at the Central Switch?

Greetings Nastra

**Comment by Nastras** [comment]
![D17FE3D2-FAAC-4967-9B5D-454577C276A8](https://user-images.githubusercontent.com/27686185/75513502-553ad500-59f5-11ea-99d2-342f0f6f1367.jpeg)

**Comment by Nastras** [comment]
Here is the body to call the new home mode via the API:
![15BA5270-E6D2-42C9-8447-B2AF87CC400F](https://user-images.githubusercontent.com/27686185/77665010-4fe29300-6f7f-11ea-8d94-25ed2df0b5c5.jpeg)

**Comment by MaurizioB84** [comment]
I think this function it's present only with version 2 of the tado devices, with the new one we need to pay to have it.

For this reason I created an automation in order to change the temperature of all devices in manual mode.

If is it possible to expose a switch to change from home to away and viceversa is very useful

thanks

**Comment by itavero** [comment]
v2 in the URL only says something about the API version. I have not tried the API call itself after the manual button was added in the app, but in the past I did an attempt at making an automation for this myself, see: https://github.com/itavero/homebridge-tado-presence

I have a V3+ thermostat and bridge and in my experiment I used the `api/v2/homes/XXX/presence` endpoint to change the presence, which worked without any issue, so I'm pretty sure the `api/v2/homes/XXX/presenceLock` endpoint will also work with newer devices.
Note that I do not have the subscription, so I don't have the "auto" option.

I might find some time in the near future to decompile the APK (for Android) again, and see if I can find any more specifics around this API call.
